### PR TITLE
Updated installation guide

### PR DESCRIPTION
**Pull request by Kalanamith**
Update on how to setup environment variables

### CODE REVIEW

File: install-guide.md [hunk 1]
```diff
@@ -234,9 +234,16 @@ Copy the configuration files `<testnet or mainnet>-core-config.json` and `<testn
 
 Start a new console and run the following command to start the core service. 
 
+On Windows
 ```
-ION_CORE_CONFIG_FILE_PATH=/usr/local/src/ion/json/testnet-core-config.json
-ION_CORE_VERSIONING_CONFIG_FILE_PATH=/usr/local/src/ion/json/testnet-core-versioning.json
+$Env:ION_CORE_CONFIG_FILE_PATH="\usr\local\src\ion\json\testnet-core-config.json"
+$Env:ION_CORE_VERSIONING_CONFIG_FILE_PATH="\usr\local\src\ion\json\testnet-core-versioning.json"
```
Comment (thehenrytsai) [inline install-guide.md:240]: Can you:

1. Qualify that the command is for Windows PowerShell.
1. Add the same for running ION Bitcoin microservice? (Line 219 ~ 220)

**Review by thehenrytsai — CHANGES_REQUESTED** [review]

File: install-guide.md (hunk 1 above)
Comment (Kalanamith) [reply install-guide.md:240]: Alright, will do 
Cheers

**Comment by thehenrytsai** [comment]
@Kalanamith, gentle ping.

**Comment by thehenrytsai** [comment]
Closing PR due to ~100 days of inactivity. Feel free to reopen whenever.

**Pull request closed without merging**